### ISSUE TITLE
How can I use adapters when each input has multiple [cls] tokens

### ISSUE THREAD

**Issue by sbassam** [Adapter-Hub/adapter-transformers#99]
Hi!

I'm trying to use adapters for a task where the input is a set of documents **each containing multiple sentences (hence multiple `cls` tokens)**. The goal is to assign a binary label to each sentence of each document. In the case of fine-tuning, I would get the output of a pretrained `BERTModel`, grab the cls embeddings and feed it through a simple classifier. But with adapters, I'm not sure how I can access the output of the base language model, process it, and feed it to a classification head. Any pointers would be greatly appreciated!

**Comment by calpt** [comment]
Hi @sbassam,

There should be no difference in accessing the output of the base language model between full fine-tuning and adapters. You can use the `BertModel` class, add new adapters, enable adapter training and then access the CLS tokens the same way in every forward pass. You should also be able to create a class that adds a custom prediction head in the same style as for full fine-tuning (e.g. `BertForSequenceClassification`).
